[2023-09-13T07:38:43.584+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-02T00:00:00+00:00 [queued]>
[2023-09-13T07:38:43.614+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-02T00:00:00+00:00 [queued]>
[2023-09-13T07:38:43.615+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:43.668+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-02 00:00:00+00:00
[2023-09-13T07:38:43.674+0000] {standard_task_runner.py:57} INFO - Started process 131 to run task
[2023-09-13T07:38:43.696+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-02T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmps_v3l6a4']
[2023-09-13T07:38:43.702+0000] {standard_task_runner.py:85} INFO - Job 32: Subtask extract_data
[2023-09-13T07:38:43.831+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-02T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:44.101+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-02T00:00:00+00:00'
[2023-09-13T07:38:44.553+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:44.597+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230902T000000, start_date=20230913T073843, end_date=20230913T073844
[2023-09-13T07:38:44.671+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:44.700+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:19.038+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-02T00:00:00+00:00 [queued]>
[2023-09-15T11:31:19.074+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-02T00:00:00+00:00 [queued]>
[2023-09-15T11:31:19.075+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:19.111+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-02 00:00:00+00:00
[2023-09-15T11:31:19.118+0000] {standard_task_runner.py:57} INFO - Started process 368 to run task
[2023-09-15T11:31:19.158+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-02T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp6_8cxw52']
[2023-09-15T11:31:19.175+0000] {standard_task_runner.py:85} INFO - Job 24: Subtask extract_data
[2023-09-15T11:31:19.348+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-02T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:19.534+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-02T00:00:00+00:00'
[2023-09-15T11:31:19.874+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:19.892+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230902T000000, start_date=20230915T113119, end_date=20230915T113119
[2023-09-15T11:31:19.967+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:19.993+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
